carla_integration/SubsystemControllerParams.yaml

```diff
@@ -49,9 +49,14 @@ guidance:
         - /guidance/plugins/sci_strategic_plugin
         - /guidance/plugins/stop_controlled_intersection_tactical_plugin
         - /guidance/plugins/platoon_control_ihp
-        - /guidance/plugins/platoon_strategic_ihp
+        - /guidance/plugins/platoon_strategic_ihp_node
         - /guidance/plugins/platooning_tactical_plugin
         - /guidance/plugins/yield_plugin
+      ros2_initial_plugins:
```

Comment:
Whats the point of adding this field if all of these plugins are already included in required and auto_activated?

Reply:
It distinguishes ros1 and ros2 plugins. It's required by the plugin_manager after Misheel found the automatic detection could never be made to work at startup so it has to be specified. required and auto_activated have different meanings.

Reply:
right, that makes sense